dom-event-delegation/main.js

```diff
@@ -0,0 +1,12 @@
+const $taskList = document.querySelector('.task-list');
+$taskList.addEventListener('click', handleTaskList);
+
+function handleTaskList(event) {
+  console.log('event.target', event.target);
+  console.log('event.target.tagName', event.target.tagName);
+  if (event.target && event.target.tagName === 'BUTTON') {
+    const closestItem = event.target.closest('.task-list-item');
```

Comment:
Needs the `$` in front of the variable name! You're storing a DOM element, so should include the `$`